### PR TITLE
move media admin translation

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: MediaAdminBundle/Resources/config/leftpanel.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    open_orchestra_backoffice.left_panel.folder:
-        class: %open_orchestra_backoffice.left_panel.folder.class%
+    open_orchestra_mediaadmin.left_panel.folder:
+        class: %open_orchestra_mediaadmin.left_panel.folder.class%
         arguments:
             - @open_orchestra_media.repository.media_folder
         tags:
-            - { name: open_orchestra_backoffice.left_panel.strategy }
\ No newline at end of file
+            - { name: open_orchestra_mediaadmin.left_panel.strategy }
```
Comment (nicolasThal) [inline MediaAdminBundle/Resources/config/leftpanel.yml]: eof

File: MediaAdminBundle/Resources/views/Tree/tree.html.twig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% import "OpenOrchestraMediaAdminBundle:Tree:tree-macros.html.twig" as macros %}
```
Comment (nicolasThal) [inline MediaAdminBundle/Resources/views/Tree/tree.html.twig]: If this file is the same that the one used in the backofficeBundle, you don't have to override it

File: MediaAdminBundle/Resources/views/Tree/showFolderTree.html.twig [hunk 3]
```diff
@@ -1,6 +1,6 @@
-{% extends 'OpenOrchestraBackofficeBundle:Tree:tree.html.twig' %}
+{% extends 'OpenOrchestraMediaAdminBundle:Tree:tree.html.twig' %}
```
Comment (nicolasThal) [inline MediaAdminBundle/Resources/views/Tree/showFolderTree.html.twig]: use the old file

File: MediaAdminBundle/Resources/views/Tree/tree.html.twig (hunk 2 above)
Comment (itkg-nanne) [inline MediaAdminBundle/Resources/views/Tree/tree.html.twig]: tree-macro is not the same in backoffice and mediaadmin.

Comment (itkg-nanne) [inline MediaAdminBundle/Resources/views/Tree/tree.html.twig]: Maybe you speak about tree.html.twig. I duplicate this file in order to controller of mediaAdmin calling twig in mediaAdmin. @nicolasThal

Comment (nicolasThal) [inline MediaAdminBundle/Resources/views/Tree/tree.html.twig]: I read it too fast.

Sorry, this is good for me

**Comment by nicolasThal** [comment]
:+1: I read it a little too quickly

File: MediaAdminBundle/Resources/views/Tree/tree.html.twig (hunk 2 above)
Comment (itkg-nanne) [inline MediaAdminBundle/Resources/views/Tree/tree.html.twig]: in tree.html, there is the import of mediaAdmin/tree-macro, @nicolasThal

**Pull request merged**